src/components/StopWatch.jsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useRef, useState } from "react";
+import Pause from "../assets/pause.png";
+import Play from "../assets/play.png";
+import Restart from "../assets/restart.png";
+import Stop from "../assets/stop.png";
+import PauseDark from "../assets/pauseDark.png";
+import PlayDark from "../assets/playDark.png";
+import RestartDark from "../assets/restartDark.png";
+import StopDark from "../assets/stopDark.png";
+import { useTasks } from "../context/task-context";
+import { useTheme } from "../context/theme-context";
+
+export const StopWatch = ({ duration, taskId, disabled }) => {
+  const { completeTask } = useTasks();
+  const { theme } = useTheme();
+  const {dark} = theme;
```

Comment:
you can destructure it in one line, {theme: {dark}} = useTheme()

Reply:
I didn't know that, thanks for letting me know